server/locale/DE/Pi30/mutations/pi30.js

```diff
@@ -0,0 +1,205 @@
+import Logo from '../../../../message/logos';
+import {
+    xSmallFallback,
+    textWrap,
+    messageLogoWidth,
+    altNoWrap,
+    setLogoTop,
+    xSmallNoWrap,
+    addPeriod
+} from '../../../../message/mediaQueries';
+import { flexLogoMutations, textLogoMutations } from '../../../../message/logoMutations';
+
+const headlineBreaks = [
+    {
+        sizes: ['xsmall'],
+        breaks: ['kaufen,']
+    },
+    {
+        sizes: ['medium'],
+        breaks: ['30']
```

Comment:
```suggestion
        breaks: ['nach']
```
Let's actually change this to be just 'nach' because the numbers have a tendency to change more often than other words.